lib/process-services/comments/comment-list.component.spec.ts

```diff
@@ -133,4 +133,9 @@ describe('CommentListComponent', () => {
         expect(elements[0].innerText).toContain(commentList.getUserShortName(testComment.createdBy));
         expect(fixture.nativeElement.querySelector('#comment-user-icon:empty')).toBeNull();
     });
+
+    it('should hide the datatable header in comment-list as showHeader is false', () => {
+        fixture.detectChanges();
+        expect(fixture.nativeElement.querySelector('.adf-datatable-header')).toBe(null);
```

Comment:
can we use the whenStable like 

        fixture.detectChanges();

        fixture.whenStable().then(() => {
            fixture.detectChanges();
            expect(element.querySelector('.adf-datatable-header')).toBe(null);
            done();
        });